authentication/admin.py

```
@@ -1,3 +1,23 @@
 from django.contrib import admin
+from .models import OTPVerification
+from django.contrib.admin.actions import delete_selected
```

Ruff (0.8.2)

`django.contrib.admin.actions.delete_selected` imported but unused

Remove unused import: `django.contrib.admin.actions.delete_selected`

(F401)